dht.go

```diff
@@ -400,3 +402,18 @@ func (dht *IpfsDHT) protocolStrs() []string {
 func mkDsKey(s string) ds.Key {
 	return ds.NewKey(base32.RawStdEncoding.EncodeToString([]byte(s)))
 }
+
+func (dht *IpfsDHT) PeerId() peer.ID {
```

Comment:
This is redundant. We already have `host.ID()`.

Reply:
See [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter). It requires knowledge of the relationship between host and the DHT instance to know that it's the correct ID. This method removes all doubt, and will track API changes.

Reply:
Ok, I take this argument, as it's nice for DX ;-) nit - just one tiny change to be idiomatic:

```suggestion
func (dht *IpfsDHT) PeerID() peer.ID {
```